### DISCUSSION TITLE
Show exact property name in Inspector - option?

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9867] [Editor]
How about an option in project settings to show the exact property names as you would type them into GDscript in the inspector panel instead of the nice names currently shown in the inspector panel?

For example:

"stretch_mode" instead of "Stretch Mode"?

**Comment by theraot** [comment]
Select "Raw" here:

![image](https://github.com/godotengine/godot-proposals/assets/1212509/d401b6d7-b8ab-49d2-a2dd-0c70679ed333)

**Reply by shak2** [reply]
Nice! But that could really do with a more descriptive name...

**Reply by Calinou** [reply]
I think **Raw** is as close as it gets, although there could be some sort of preview with an example.

For instance:

- Raw (like_this)
- Capitalized (Like This)
- Localized (Like This), with "Like This" localized in the user's language.

I opened a pull request for this: https://github.com/godotengine/godot/pull/92665

**Comment by akien-mga** [comment]
Fixed by https://github.com/godotengine/godot/pull/92665.
